### DISCUSSION TITLE
최신버전 설치 관련

### DISCUSSION THREAD

**Discussion by presso22** [changdoc/naver-vaccine-macro#270] [Q&A]
크롬 확장 삭제하고, 최신버전 다운로드 하여! 설치하려고 하니
(#29 의 https://trend21c.tistory.com/1030 확인중)

저는 개발자모드에 + 표시가 없어서 
방법이 이게 맞는지 모르겠습니다.
![개발자](https://user-images.githubusercontent.com/88812138/129182121-077269d9-2a75-4ff6-9a44-b503876f69c2.png)

**Comment by changdoc** [comment]
네 저기서 "압축해제된 확장 프로그램을 로드합니다"로 다운 받으신 zip압축 해제해둔 폴더를 선택하시면 설치 될 거에요.

**Reply by presso22** [reply]
감사합니다!!!!
